_inc/client/plans/single-product-backup.jsx

```diff
@@ -0,0 +1,188 @@
+
+export function SingleProductBackup( props ) {
+	const { products, siteRawUrl } = props;
+
+	const backupPlanPrices = {
+		jetpack_backup_daily: {
+			monthly: products.jetpack_backup_daily_monthly.cost,
+			yearly: products.jetpack_backup_daily.cost,
+		},
+		jetpack_backup_realtime: {
+			monthly: products.jetpack_backup_realtime_monthly.cost,
+			yearly: products.jetpack_backup_realtime.cost,
+		},
+	};
+
+	const currencySymbol = getCurrencyDefaults( products.jetpack_backup_daily.currency_code ).symbol;
+
+	const [ selectedBackupType, setSelectedBackupType ] = useState( 'real-time' );
+
+	return (
+		<React.Fragment>
+			<h1 className="plans-section__header">{ __( 'Single Products' ) }</h1>
+			<h2 className="plans-section__subheader">
+				{ __( "Just looking for backups? We've got you covered." ) }
+			</h2>
+			<div className="single-product-backup__accented-card-container">
+				<div className="single-product-backup__accented-card">
+					<div className={ 'single-product-backup__accented-card__header' }>
+						<SingleProductBackupHeader
+							currencySymbol={ currencySymbol }
+							backupPlanPrices={ backupPlanPrices }
+						/>
+					</div>
+					<div className={ 'single-product-backup__accented-card__body' }>
+						<SingleProductBackupBody
+							currencySymbol={ currencySymbol }
+							selectedBackupType={ selectedBackupType }
+							setSelectedBackupType={ setSelectedBackupType }
+							backupPlanPrices={ backupPlanPrices }
+							siteRawUrl={ siteRawUrl }
+						/>
+					</div>
+				</div>
+			</div>
+		</React.Fragment>
+	);
+}
+
+function SingleProductBackupHeader( props ) {
+	const { currencySymbol, backupPlanPrices } = props;
+
+	return (
+		<div className="single-product-backup__header-container">
+			<h3>{ __( 'Jetpack Backup' ) }</h3>
+			<PlanPriceDisplay currencySymbol={ currencySymbol } backupPlanPrices={ backupPlanPrices } />
+		</div>
+	);
+}
+
+export function PlanPriceDisplay( props ) {
+	const { backupPlanPrices, currencySymbol } = props;
+	const perYearPriceRange = `${ currencySymbol }${ backupPlanPrices.jetpack_backup_daily.yearly }-${ backupPlanPrices.jetpack_backup_realtime.yearly } /year`;
+
+	return (
+		<div className="single-product-backup__plan-price-display-container">
+			<div className="slashed-price__container">
+				<div className="slashed-price__slash"></div>
+				<div className="slashed-price__price">{ `${ currencySymbol }${ backupPlanPrices
+					.jetpack_backup_daily.monthly * 12 }-${ backupPlanPrices.jetpack_backup_realtime.monthly *
+					12 }` }</div>
+			</div>
+			<div className="plans-price__container">
+				<span className="plans-price__span">{ perYearPriceRange }</span>
+			</div>
+		</div>
+	);
+}
+
+function SingleProductBackupBody( props ) {
+	const {
+		currencySymbol,
+		backupPlanPrices,
+		selectedBackupType,
+		setSelectedBackupType,
+		siteRawUrl,
+	} = props;
+
+	const upgradeLinks = {
+		'real-time': `https://wordpress.com/checkout/${ siteRawUrl }/professional`,
```

Comment:
It has been updated to use a Jetpack redirect, although it will not work until I do a small patch in wpcom to make the redirect point to the correct place.